llvm/unittests/Support/DebugLogTest.cpp

```diff
@@ -128,6 +128,131 @@ TEST(DebugLogTest, DestructorPrefix) {
   // After destructors, nothing should have been printed.
   EXPECT_EQ(os.str(), "");
 }
+
+TEST(DebugLogTest, LDBG_MACROS) {
```

Comment:
I would split this into multiple smaller test functions. That shouldn't impact test coverage, but would definitely make triaging any potential failures easier :)